packages/reports/src/app/addRoutesToApp.ts

```diff
@@ -4,10 +4,16 @@
  */
 
 import { fetchReport } from '../routes';
+import { authenticationMiddleware } from '../auth';
 
 export function addRoutesToApp(app) {
   /**
-   * GET routes
-   **/
-  app.get('(/v[0-9]+)?/fetchReport/:reportCode', fetchReport);
+   * Attach authentication to each endpoint
+   */
+  app.use(authenticationMiddleware);
+
+  /**
+   * POST routes
+   */
+  app.post('(/v[0-9]+)?/fetchReport/:reportCode', fetchReport);
```

Comment:
Switched to `post` for this as we want to pass username/pass in the body. Not a bad idea anyway, as we may end up with a more complex request format anyway